docs/application/native/guides/connectivity/zigbee.md

```diff
@@ -0,0 +1,315 @@
+
+4. Call the `zb_create()` function to create zigbee service handle:
+
+   ```
+   int zb_create(zb_zigbee_h *handle);
+   ```
+
+5. Call the `zb_destroy()` function to destroy zigbee service handle:
+
+   ```
+   int zb_destroy(zb_zigbee_h handle);
+   ```
+
+## Enable or Disable Zigbee Device
+
+1. To enable Zigbee, you must call `zb_enable()` API and wait for Zigbee state changed event `ZB_ZDP_ENABLE_EVENT`.
+
+   ```
+   static void _zb_event_cb(zb_nwk_addr addr16, zb_ieee_addr addr64, zb_event_e e,
+                            zb_event_data_h ev, void* user_data)
+   {
+       // Handle e for ZB_ZDP_ENABLE_EVENT
+   }
+
+   zb_zigbee_h enable_zigbee_device(void)
+   {
+       zb_zigbee_h handle;
+       int ret = ZIGBEE_ERROR_NONE;
+
+       ret = zb_create(&handle);
+       if (ZIGBEE_ERROR_NONE != ret) {
+           printf("zb_create() - FAILED!!!");
+           return NULL;
+       }
+
+       ret = zb_set_event_cb(handle, _zb_event_cb);
+       if (ZIGBEE_ERROR_NONE != ret) {
+           printf("zb_set_event_cb() - FAILED!!!");
+           return NULL;
+       }
+
+       ret = zb_enable(handle);
+       if (ZIGBEE_ERROR_NONE != ret) {
+           printf("zb_enable() - FAILED!!!");
+           return NULL;
+       }
+
+       return handle;
+   }
+   ```
+
+2. To disable Zigbee, you must call `zb_disable()` API and wait for Zigbee state changed event `ZB_ZDP_ENABLE_EVENT`.
+
+   ```
+   static void _zb_event_cb(zb_nwk_addr addr16, zb_ieee_addr addr64, zb_event_e e,
+                            zb_event_data_h ev, void* user_data)
+   {
+       // Handle e for ZB_ZDP_ENABLE_EVENT
+   }
+
+   int disable_zigbee_device(zb_zigbee_h handle)
+   {
+       int ret = ZIGBEE_ERROR_NONE;
+
+       /*
+        * It is expected that user has already registered zb_set_event_cb(handle, _zb_event_cb),
+        * because the actual result proving that zigbee device has been disabled
+        * will come in _zb_event_cb().
+        */
+
+       ret = zb_disable(handle);
+       if (ZIGBEE_ERROR_NONE != ret) {
+           printf("zb_disable() - FAILED!!!");
+           return FALSE;
+       }
+
+       return TRUE;
+   }
+   ```
+
+## Form or Disable Zigbee Network Coordinator
+
+1. To form a zigbee network coordinator, you must call `zb_form_network()` API and wait for result in `zb_form_network_cb` callback.
+
+   ```
+   static void _zb_form_network_done_cb(zb_nwk_addr panid, void *user_data)
+   {
+       printf("form_network_done received PANID = 0x%04X\n", panid);
+   }
+
+   int run_form_network(zb_zigbee_h handle)
+   {
+       int ret = ZIGBEE_ERROR_NONE;
+
+       ret = zb_form_network(handle, _zb_form_network_done_cb, NULL);
+       if (ZIGBEE_ERROR_NONE != ret) {
+               msg("zb_form_network() - FAILED!!!");
+               return FALSE;
+       }
+
+       return TRUE;
+   }
+   ```
+
+2. To disable a zigbee network coordinator, you must call `run_disable_network()` API and wait for result in `zb_disable_network_cb` callback.
```

Comment:
Just for check : `run_disable_network()`? or `zb_disable_network()`?

Reply:
Thanks, resolved comment.